librz/analysis/p/analysis_6502.c

```diff
@@ -45,114 +47,177 @@ static void _6502_analysis_update_flags(RzAnalysisOp *op, int flags) {
 }
 
 /* ORA, AND, EOR, ADC, STA, LDA, CMP and SBC share this pattern */
-static void _6502_analysis_esil_get_addr_pattern1(RzAnalysisOp *op, const ut8 *data, int len, char *addrbuf, int addrsize) {
+static void _6502_analysis_esil_get_addr_pattern1(RzAnalysisOp *op, const ut8 *data, int len,
```

Comment:
remove that `int` and make it `ut32` or `size_t`

Reply:
`size_t` is good

Reply:
Regarding grouping I think it would only make sense to group `RZ_NULLABLE char *esiladdr_out, int esiladdr_size,`, similarly to `_6502ILAddr`, but I would just leave it since 6502 esil is deprecated by this anyway.